packages/shared/src/components/AddImagesButton.tsx

```diff
@@ -2,25 +2,31 @@
 
 import {
     ActionMenu,
+    Flyout,
     IconArrowCircleUp20,
     IconImageStack20,
     LoadingCircle,
     MenuItemContentSize,
 } from '@frontify/fondue';
-import React, { useEffect, useRef, useState } from 'react';
+import React, { useRef, useState } from 'react';
 import { joinClassNames } from '../utilities/react/joinClassNames';
 
 export type AddImagesButtonProps = {
     isLoading?: boolean;
     label: string;
     secondaryLabel?: string;
     icon?: JSX.Element;
-    onDrop: (files: FileList) => void;
+    onDrop?: (files: FileList) => void;
     fillParentContainer?: boolean;
     onUploadClick: () => void;
     onAssetChooseClick: () => void;
+    setIsMenuOpen?: (isOpen: boolean) => void;
```

Comment:
maybe rename it to something more generic? `onOpenChange` ?

Reply:
true. I removed it an used the onClick which is anyway needed for the ones where the openMenu is not necessary.